### DISCUSSION TITLE
遗失的数据：《开源之迷》附录A - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#21] [Announcements]
# 遗失的数据：《开源之迷》附录A - 开源之道

https://opensourceway.community/posts/book-of-open-source/the-open-source-trilogy/the-fascinating-of-open-source-appendix-a/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

所以, 下次重印, 必须是全彩了, 否则这么多好容易制作出来的可视化图表, 无法 got 到精彩,
其实, 每个一个结论,都有其历史变迁故事值得挖掘,比如:

2019 
> ...95%的人认为企业级开源非常之重要！

那么 2009?1999?1989?
不同时代, 也是这么认为的嘛? 当然不可能, 为什么? 是什么引发了变化?
可以加强嘛? 这一变化导制了什么?
